com.unity.ml-agents/Runtime/Sensor/VectorSensor.cs

```diff
@@ -1,7 +1,8 @@
 using System.Collections.Generic;
 using UnityEngine;
+using MLAgents.Sensor;
 
-namespace MLAgents.Sensor
+namespace MLAgents
```

Comment:
What's the reason for this change? Should we (roughly) keep the namespaces 1:1 with the subdirectories?

Reply:
Yes but now the user would have to import both MLAgents and MLAgents.Sensors when implementing an Agent that has a CollectObservation implementation

Reply:
All right. Removed the `MLAgents.Sensor` namespace alltogether